gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/testing/LocalResourceManagerHelper.java

```diff
@@ -550,17 +551,21 @@ private LocalResourceManagerHelper() {
   }
 
   /**
-   * Creates a LocalResourceManagerHelper object that listens to requests on the local machine.
+   * Creates a {@code LocalResourceManagerHelper} object that listens to requests on the local
+   * machine.
    */
   public static LocalResourceManagerHelper create() {
     return new LocalResourceManagerHelper();
   }
 
   /**
-   * Returns a ResourceManagerOptions instance that sets the host to use the mock server.
+   * Returns a {@link ResourceManagerOptions} instance that sets the host to use the mock server.
```

Comment:
Nit: links to self in javadoc are not useful, we could use `{@code LocalResourceManagerHelper}` instead.